CHANGES.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 - [NEW] Constructor for `Database` subclasses.
 - [IMPROVED] Documentation for `Database.findByIndex` to show complete selector.
+- [FIX] `CouchDbException: 400 Bad Request: bad_request: invalid_json` when a
+  query parameter contains a semicolon.
```

Comment:
Technically the fix is for any of `=+&;` but I don't know if you want to go to that level of detail